test/UnitTest.cs

```diff
@@ -297,10 +302,6 @@ public void TestJobench()
             string write_dir_fn = $"../../../../test/regress/output/jobench";
             string expect_dir_fn = $"../../../../test/regress/expect/jobench";
 
-            // make sure all queries can generate phase one opt plan
-            QueryOption option = new QueryOption();
-            option.optimize_.TurnOnAllOptimizations();
```

Comment:
Please keep this line as it turns on all optimizations for jobench run.

Reply:
I think this option is not used, the option has optimize turned on in psql/Program.cs line 31

Reply:
You are right. In a separate patch, can you please remove psql project and move its content to unittest? We used to have psql designed as a SQL client but we don't think we will need it anymore.

Reply:
got it